agent/agent_installer/Windows/AgentService.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+<id>Hydra Lab Agent Service</id>
+<name>Hydra Lab Agent Service</name>
+<description>Hydra Lab Agent Service.</description>
+
+<executable>java</executable>
+<arguments>-Xms1024m -Xmx2048m -jar agent.jar</arguments>
+<startmode>Automatic</startmode>
+<logpath> {LOG_FILE_LOCATION} </logpath>
```

Comment:
Regarding this, I guess your idea is to let the user update this, right? Please make it clear in wiki.

Approved with suggestions.

Reply:
And could we reindent this file?

Reply:
I'm not sure if actionable, but could we further automate these 2 steps?
<img width="936" alt="image" src="https://user-images.githubusercontent.com/121116183/212783283-4ab1586e-b714-4268-892f-85bc96cf5c45.png">

Like we could create a new install.bat/install.ps with the following pseudo-code:

```bash
$currentPath=Get-Current-Absolute-Path
Replace-Text-In-File AgentService.xml '{LOG_FILE_LOCATION}' "$currentPath\service-log"
.\AgentService.exe install
```

So that all the user need to do is to run the install.bat with admin permission.

Reply:
I'm not sure if actionable, but could we further automate these 2 steps?
<img width="936" alt="image" src="https://user-images.githubusercontent.com/121116183/212783283-4ab1586e-b714-4268-892f-85bc96cf5c45.png">

Like we could create a new install.bat/install.ps with the following pseudo-code:

```bash
$currentPath=Get-Current-Absolute-Path
Replace-Text-In-File AgentService.xml '{LOG_FILE_LOCATION}' "$currentPath\service-log"
.\AgentService.exe install
```

So that all the user need to do is to run the install.bat with admin permission.